tests/api/test_projects.py

```diff
@@ -53,36 +58,134 @@ def test_validtion_error(self, flask_api_url, pytestconfig):
                                  headers={'Content-Type': 'application/json'})
         print(response.text)
 
-    def test_missformed_json_error(self, flask_api_url):
-        assert True
+    def test_missformed_json_error(self, flask_api_url, pytestconfig):
+        """Test both invalid json and for json which does not match,
+        test for Bad Request and Validation Error.
+        """
+        invalid_manifest = os.path.join(
+            str(pytestconfig.rootdir),
+            'tests',
+            'testmanifests',
+            'invalidexample0.json'
+        )
+        unmatching_manifest = os.path.join(
+            str(pytestconfig.rootdir),
+            'tests',
+            'testmanifests',
+            'invalidexample1.json'
+        )
+        with open(invalid_manifest, 'r', encoding='utf-8') as tf:
+            invalid_data = str(tf.read().replace('\n', ''))
+        invalid_response = requests.post(flask_api_url + "/api/projects",
+                                         data=invalid_data.encode('utf-8'),
+                                         headers={'Content-Type': 'application/json5'})
+        print(invalid_response.text)
+
+        with open(unmatching_manifest, 'r', encoding='utf-8') as tf:
+            unmatching_data = str(tf.read().replace('\n', ''))
+        unmatching_response = requests.post(flask_api_url + "/api/projects",
+                                            data=unmatching_data.encode('utf-8'),
+                                            headers={'Content-Type': 'application/json5'})
+        print(unmatching_response.text)
+
+        assert invalid_response.status_code == 400
+        assert 'Validation Error' in unmatching_response.text
+
+    def test_misformed_content_type(self, flask_api_url, pytestconfig):
+        """Test if wrong content-type header with valid json will raise error.
+        """
+        test_manifest = os.path.join(
+            str(pytestconfig.rootdir),
+            'tests',
+            'testmanifests',
+            'validexample0.json'
+        )
+        with open(test_manifest, 'r', encoding='utf-8') as tf:
+            data = str(tf.read().replace('\n', ''))
+        response = requests.post(flask_api_url + "/api/projects", data=data.encode('utf-8'),
+                                 headers={'Content-Type': 'image/gif'})
+        print(response.text)
+        assert response.status_code == 400
 
-    def test_missformed_content_type(self, flask_api_url):
-        assert True
 
 
 class TestDELETE(object):
+
     def test_unknown_id(self, flask_api_url):
-        assert True
+        """ Test for 404 when a project with unknown ID is to be deleted.
+        """
+        invalid_id = "invalid"
```

Comment:
It would be nice to a valid format here. You can just generate a new v4 uuid, a collision is (neither) impossible.

The more I think about it, this seems like to separate tests. Having "unknown" and "invalid" id checks would be nice, thanks for pointing out that issue.